[CSE-687] Office Hours (#2851)
astronomer/docs: astro/office-hours.md

---
title: 'Book Astro office hours'
navTitle: 'Office hours'
id: astro-office-hours
description: Attend office hours with Astronomer data engineers to ask non-support questions about Airflow.
---

Office hours are a way for you to meet with Astronomer data engineers and ask about Astro and Airflow features and best practices . Office hours are available weekly on a first-come, first-served basis. An office hour session is typically 30 minutes long. 

:::caution

Office hours are not a replacement for support. [Submit a support request](astro-support.md) if you have any urgent issue, bug, or something that needs fixing. Astronomer data engineers will not be able to answer support requests during office hour sessions.

If you have questions about your Astronomer account or billing, email `success@astronomer.io` instead of attending office hours. If you need more extensive assistance, guidance, or regular long-term engagement, reach out to the [Professional Services](https://www.astronomer.io/professional-services/) team.

:::

> Before an office hour appointment, please browse the documentation, [Astronomer Academy](https://academy.astronomer.io/), [Airflow Guides](https://docs.astronomer.io/learn/), or discussion on the [Astronomer Forum](https://forum.astronomer.io) to see if there are existing guides or resources that cover what you'd like to discuss. 

Reviewer: ```suggestion
```

During an office hours meeting, you can:

- Ask questions about Airflow features.
- Discuss new Astro features or provide feedback.
- Get insight on integrating Airflow with new systems.
- Do an architecture review or discuss scaling Airflow and Astro Deployments .
- Conduct a code review for a DAG.

## Book an office hours appointment 

> 

Reviewer: ```suggestion

Before booking an office hour appointment, please browse the documentation, [Astronomer Academy](https://academy.astronomer.io/), [Airflow Guides](https://docs.astronomer.io/learn/), or discussion on the [Astronomer Forum](https://forum.astronomer.io) to see if there are existing guides or resources that answer your question. 

```

---

Office hours are a way for you to meet with Astronomer data engineers and ask about Astro and Airflow features and best practices . Office hours are available weekly on a first-come, first-served basis. An office hour session is typically 30 minutes long. 

:::caution

Office hours are not a replacement for support. [Submit a support request](astro-support.md) if you have any urgent issue, bug, or something that needs fixing. Astronomer data engineers will not be able to answer support requests during office hour sessions.

If you have questions about your Astronomer account or billing, email `success@astronomer.io` instead of attending office hours. If you need more extensive assistance, guidance, or regular long-term engagement, reach out to the [Professional Services](https://www.astronomer.io/professional-services/) team.

:::

During an office hours meeting, you can:

- Ask questions about Airflow features.
- Discuss new Astro features or provide feedback.
- Get insight on integrating Airflow with new systems.
- Do an architecture review or discuss scaling Airflow and Astro Deployments .
- Conduct a code review for a DAG.

## Book an office hours appointment 

> Before booking an office hour appointment, please browse the documentation, [Astronomer Academy](https://academy.astronomer.io/), [Airflow Guides](https://docs.astronomer.io/learn/), or discussion on the [Astronomer Forum](https://forum.astronomer.io) to see if there are existing guides or resources that answer your question. 

Reviewer: ```suggestion
Before you book an office hour appointment, please browse the documentation, [Astronomer Academy](https://academy.astronomer.io/), [Airflow Guides](https://docs.astronomer.io/learn/), or discussion on the [Astronomer Forum](https://forum.astronomer.io) to see if there are existing guides or resources that answer your question. 
```
